### PR TITLE
Pr/layout kit

### PR DESCRIPTION
**Pull request by attitude**
This change is [Reviewable](https://reviewable.io/reviews/contember/interface/560)

### CODE REVIEW

File: packages/admin/src/components/pageRouting/pageComponents/GenericPage.tsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ export type GenericPageProps =
  * ```
  *
  * @group Pages
+ * @deprecated No longer used
```
Comment (matej21) [inline packages/admin/src/components/pageRouting/pageComponents/GenericPage.tsx:23]: we should either deprecate all pages, or none. I'm for none for now.

File: build/api/layout.api.md [hunk 2]
```diff
@@ -539,23 +961,16 @@ export type SafeAreaInsetsProviderProps = {
     insets?: Partial<ContainerInsets>;
 };
 
+// Warning: (ae-forgotten-export) The symbol "ScreenInsetsToCSSCustomProperties" needs to be exported by the entry point index.d.ts
```
Comment (matej21) [inline build/api/layout.api.md:964]: x

File: packages/layout/src/index.ts [hunk 3]
```diff
@@ -1,10 +1,13 @@
-export * from './directives'
+import * as Directives from './directives'
+import * as Primitives from './primitives'
```
Comment (matej21) [inline packages/layout/src/index.ts:2]: LayoutPrimitives

File: build/api/layout.api.md (hunk 2 above)
Comment (attitude) [reply build/api/layout.api.md]: Done

File: packages/admin/src/components/pageRouting/pageComponents/GenericPage.tsx (hunk 1 above)
Comment (attitude) [reply packages/admin/src/components/pageRouting/pageComponents/GenericPage.tsx]: Done

File: packages/layout/src/index.ts (hunk 3 above)
Comment (attitude) [reply packages/layout/src/index.ts]: Done

**Pull request merged**